client-device-connection-infinispan/pom.xml

```diff
@@ -59,18 +66,31 @@
        </exclusion>
       </exclusions>
     </dependency>
+
     <dependency>
       <groupId>org.eclipse.hono</groupId>
-      <artifactId>hono-client-common</artifactId>
+      <artifactId>hono-core</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-client-common</artifactId>
+    </dependency>
+
+    <dependency> <!-- TODO: changed from vertx-web because there only seems to be dependency on -core. Maybe there are runtime dependencies though??? -->
```

Comment:
@sophokles73 do you know? Is it safe to change this or do you think there might be some runtime dependencies that we need from -web?